src/rules/declaration-block-no-ignored-properties/README.md

```diff
@@ -0,0 +1,53 @@
+# declaration-block-no-ignored-properties
+
+Disallow properties has no effect.
+
+```css
+a { display: inline; width: 100px; }
+/**                        ↑
+ *                  This property */
+```
+
```

Comment:
We should probably add an "expanded description" here e.g.

Certain property value pairs will cause other property value pairs to be ignored. For example, when `display: inline` is used the `width`, `height` and `margin` properties are ignored.